website/blog/2022/07/05/use-dataant-to-monitor-apisix.md

```diff
@@ -0,0 +1,94 @@
+---
+title: "How to monitor Apache APISIX with DataAnt"
+authors:
+  - name: "Fei Han"
+    title: "Technical Writer"
+    url: "https://github.com/hf400159"
+    image_url: "https://github.com/hf400159.png"
+keywords: 
+- Apache APISIX
+- API Gateway
+- Dataant
+description: This article mainly introduces how to upload the API Gateway Apache APISIX indicator data to the DATAANT monitoring system through the DataAnt Agent.
+tags: [Authentication]
+---
+
+> This article mainly introduces how to upload the APISIX indicator data to the DATA ANT monitoring system through the DataAnt Agent.
+
+<!--truncate-->
+
+## Background
+
+[Apache APISIX](https://github.com/apache/apisix) is an open source cloud native API gateway. As an API gateway, it has the characteristics of dynamic, real-time, and high performance. It provides rich traffic management functions such as load balancing, dynamic upstream, grayscale publishing, service fusing, identity authentication and observability. You can use APISIX to handle the traditional north-south traffic and the east-west traffic between services. It can also be used as a K8s ingress controller. Thanks to the full dynamic design of APISIX, configuration changes can be made at any time without restarting the service.
+
+The DataAnt full-stack cloud monitoring system can aggregate all the operation and maintenance data of IaaS, PaaS and SaaS layers through big data and machine learning, and provide users with a unified visual interface. DataAnt allows users to move seamlessly and quickly between relevant monitoring data sources without having to switch tools and gain more granular visibility into the state of their IT systems. The DataAnt Agent it provides can monitor APISIX in real time and upload its monitoring data to the DataAnt PaSS platform to realize one-stop cloud monitoring.
+
+## Introduction
+
+![principle](https://static.apiseven.com/2022/blog/0705/1.svg)
+
+1. Collection configuration
+
+The DataAnt Agent will first initialize and register the collector through the APISIX item in `config.yaml`. The same Agent can register multiple collectors. After the collector collects the indicators exposed by APISIX, it encrypts the indicator data and uploads it to DataAnt Cloud.
+
+2. Data visualization
+
+After DataAnt Cloud receives the data, the data will be stored in the time series database after preliminary monitoring information supplementation and processing, and then APISIX can be monitored in real time through DataAnt's Dashboard.
+
+3. Alarm notification
+
+The data will also be distributed to the alarm matching processing through the message, and then the notification aggregation will be carried out. Finally, the alarm will be sent through the configured notification method, that is, the abnormal situation of APISIX can be received in real time.
```

Comment:
Wrong order, you need to use tab to indent each paragraph into each step.

<img width="850" alt="image" src="https://user-images.githubusercontent.com/36651058/178105662-293483cb-7919-43a0-b2d1-307d99881f05.png">